frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -147,24 +148,28 @@ export const PaymentInput = (): JSX.Element => {
           )
         },
         validateMin: (val) => {
+          if (minPaymentAmountCents === undefined) return true
```

Comment:
/justNoting (nothing to do): this defensive checks helps at release (new code hitting old server for env), but feels unnecessary at steady state thereafter 🤔  We should expect the client to be able to load valid env vars to operate, or things wouldn't work generally.